frontend/app/src/next/components/providers/support-chat.provider.tsx

```diff
@@ -0,0 +1,65 @@
+import useApiMeta from '@/next/hooks/use-api-meta';
+import useUser from '@/next/hooks/use-user';
+import useTenant from '@/next/hooks/use-tenant';
+import React, { PropsWithChildren, useEffect, useMemo } from 'react';
+
+interface PylonWindow extends Window {
+  pylon?: {
+    chat_settings: {
+      app_id: string;
+      email: string;
+      name: string;
+      email_hash: string;
+    };
+  };
+  Pylon?: (method: string, ...args: any[]) => void;
```

Comment:
qq: kinda weird to have both `pylon` and `Pylon`?

Reply:
good catch